Numsense.UnitTests/PortugueseExamples.fs

```diff
@@ -0,0 +1,212 @@
+[<InlineData(                                  "oitocentoseseis",        806)>]
+[<InlineData(                        "novecentosecinquentaenove",        959)>]
+[<InlineData(                                              "mil",       1000)>]
+[<InlineData(                                           "mileum",       1001)>]
+[<InlineData(                                         "miledois",       1002)>]
+[<InlineData(                              "milesessentaequatro",       1064)>]
+[<InlineData(                                          "doismil",       2000)>]
+[<InlineData(                       "oitomilsetecentosevinteeum",       8721)>]
+[<InlineData(                          "novemilcentoevinteetrês",       9123)>]
+[<InlineData(                                           "dezmil",      10000)>]
+[<InlineData(                                   "dezmilcentoeum",      10101)>]
+[<InlineData(                                 "dezmilcentoedois",      10102)>]
+[<InlineData(          "vinteetrêsmilquinhentosesessentaequatro",      23564)>]
+[<InlineData(                                   "oitentamiledez",      80010)>]
+[<InlineData(                                           "cemmil",     100000)>]
+[<InlineData(                                        "cemmileum",     100001)>]
+[<InlineData(     "trezentosedezanovemiltrezentosequarentaenove",     319349)>]
+[<InlineData(                                         "ummilhão",    1000000)>]
+[<InlineData(                                      "ummilhãoeum",    1000001)>]
+[<InlineData(                                  "doismilhõesedez",    2000010)>]
+[<InlineData(                            "seismilhõesetrintamil",    6030000)>]
+[<InlineData(      "novemilhõesduzentoseummilseiscentoseoitenta",    9201680)>]
+[<InlineData(                                       "dezmilhões",   10000000)>]
+[<InlineData(                                  "dezmilhõesenove",   10000009)>]
+[<InlineData(                                  "vinteeummilhões",   21000000)>]
+[<InlineData(
+        "quarentaesetemilhõesseiscentosesessentaeummilesessenta",   47661060)>]
+[<InlineData(                                       "cemmilhões",  100000000)>]
+[<InlineData(                            "duzentosecincomilhões",  205000000)>]
+[<InlineData(                         "trezentos-e-sete-milhões",  307000000)>]
+[<InlineData(
+         "quatrocentosesessentaeummilhõessessentamileseiscentos",  461060600)>]
+[<InlineData(                                       "milmilhões", 1000000000)>]
+[<InlineData(                                   "doismilmilhões", 2000000000)>]
+[<InlineData(
+           "doismilequarentaenovemilhõesseiscentosecinquentamil", 2049650000)>]
+[<InlineData(
+    "doismilcentoequarentaesetemilhõesquatrocentoseoitentaetrêsmilseiscentosequarentaesete",
+    System.Int32.MaxValue)>]
+let ``tryParsePortuguese returns correct result`` (portuguese, expected) =
```

Comment:
Bonus points for *not* repeating my mistake of naming the test function `tryOfPortuguese... etc.` :+1: